src/components/Inventory/index.tsx

```diff
@@ -64,15 +68,28 @@ export default function Inventory(props: InventoryProps): JSX.Element {
     history.push(appPathLocation);
   };
 
+  const importInventory = () => {
+    setImportInventoryModalOpen(true);
+  };
+
   const getEmptyState = () => {
     const emptyState = [];
 
     if (!hasSpecies) {
       emptyState.push({
-        title: strings.SPECIES,
+        title: strings.CREATE_SPECIES_LIST,
         text: emptyMessageStrings.INVENTORY_ONBOARDING_SPECIES_MSG,
         buttonText: strings.GO_TO_SPECIES,
         onClick: () => goTo(APP_PATHS.SPECIES),
+        altItem: hasNurseries
```

Comment:
for accessions, importing required user to select a seedbank as the first step.. that's why we had a check.
here, just curious how does nursery get selected for import? is it in the csv?

Reply:
There is a dropdown on the import modal and it is send on the api as a queryParameter

Reply:
Got it, saw it in vercel. Thanks.